metric/kamailio5/kamailio.cfg

```diff
@@ -410,41 +420,64 @@ route[STORE] {
 
 
         $var(homer_query_values) = "(10, '" + $utimef(%Y-%m-%d %H:%M:%S) + "', '" + $var(hep_header) +"', '" + $var(ip_header) + "', '" + $var(payload) + "', '" + $msg(buf) + "')";
-        mq_add("queue_sip_insert", "$var(table)", "$var(homer_query_values)");
+
+        if($sht(cc=>packet::count) == $null) $sht(cc=>packet::count) = 0;
+        $sht(cc=>packet::count) = $sht(cc=>packet::count) + 1;
+
+	mq_add("queue_sip_insert", "$var(table)", "$var(homer_query_values)");
 }
 
 route[SIP_INSERT] {
-        xlog("L_CRIT","Enter queue_sip_insert\n");
+	$var(current_qsize) = mq_size("queue_sip_insert");
+	if ($var(current_qsize) == -1) {
+		$var(total_qsize) = 1;
+		xlog("L_DBG","Insert count: $sht(cc=>packet::count) \n");
+		$sht(cc=>packet::count) = 0;
+	}
+	$var(total_qsize) = $var(total_qsize) + $var(current_qsize);
+        xlog("L_DBG","Enter queue_sip_insert! Current queue size: $var(current_qsize), total queue count: $var(total_qsize) \n");
         
 	$var(sql_query_statement) = "";
 
```

Comment:
We should rename route[INSERT] to route[SIP_QUEUE_INSERT]

Reply:
since the route is not handling only sip but it's now generic, maybe something like "FLUSH_QUEUE"?